worker/build.rs

```diff
@@ -78,19 +80,26 @@ fn main() {
     #[cfg(target_os = "windows")]
     {
         if !std::path::Path::new("worker/out/msys/bin/make.exe").exists() {
-            let python = if Command::new("where")
-                .arg("python3.exe")
+            let python = if env::var("PYTHON").is_ok() {
+                env::var("PYTHON").unwrap()
```

Comment:
It is anti-pattern to check `.is_ok()` just to `.unwrap()` right after this (clippy should complain about this). Use pattern matching instead:
```rust
let python = if let Ok(python) = env::var("PYTHON") {
    python
} else if {
    // ...
};
```